### ISSUE TITLE
Transfer issue/error descriptions living in list.ts to schema model

### ISSUE THREAD

**Issue by bleonar5** [psych-ds/psychds-validator#30]
# TL;DR
Certain errors/issues are currently listed in [list.ts](https://github.com/psych-ds/psychds-validator/blob/main/src/issues/list.ts) and ought to be moved to a suitable location in the [schema model](https://github.com/psych-ds/psych-DS/tree/master/schema_model)

Out of convenience relating to inherited code, I've ended up adding a number of floating issues/errors to list.ts in order to use the addNonSchemaIssue function to quickly develop necessary functionality in the validation process. Now that most key functionality is in place, and since the name of the game for this sprint is "updating schema model", it is now time to tackle this. 

By the end of the process, there should no longer be any need for list.ts or the addNonSchemaIssue function, so these will be eliminated entirely, and all errors/issues will either simply be relocated to errors.yaml or collated directly with their relevant rules. Perhaps the best setup will be to reference error codes within the rule files and catalog the errors according to their codes in errors.yaml

This is essentially a sub-issue of #25
